### PR TITLE
travis - py3 compatible

### PR DESCRIPTION
**Pull request by domdinicola**
added travis CI
added support to python3.7
added tox

### CODE REVIEW

**Comment by domdinicola** [comment]
@czue can we hook travis ci here?

File: data_api/staging/models.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 from django.contrib.postgres.fields import ArrayField, JSONField
 from django.core.exceptions import FieldDoesNotExist
 from django.db import models, transaction
+from django.utils import six
```
Comment (czue) [inline data_api/staging/models.py]: what's the difference between django's `six` and this one: https://six.readthedocs.io/?

**Comment by domdinicola** [comment]
I would need admin access to the repo to setup travis

**Comment by czue** [comment]
Ah, just changed your access. Go ahead!

File: data_api/staging/models.py (hunk 1 above)
Comment (domdinicola) [reply data_api/staging/models.py]: is a customized version of the six module.. I don't think is a big difference https://docs.djangoproject.com/en/1.11/topics/python3/

**Pull request merged**